contrib/cirrus/lib.sh

```diff
@@ -290,3 +290,32 @@ remove_packaged_podman_files() {
     # Be super extra sure and careful vs performant and completely safe
     sync && echo 3 > /proc/sys/vm/drop_caches || true
 }
+
+# Execute make localbenchmarks in $CIRRUS_WORKING_DIR/data
+# for preserving as a task artifact.
+localbenchmarks() {
+    local datadir
+    req_env_vars DISTRO_NV PODBIN_NAME PRIV_NAME TEST_ENVIRON TEST_FLAVOR
+    req_env_vars VM_IMAGE_NAME EC2_INST_TYPE
+
+    datadir=$CIRRUS_WORKING_DIR/data
+    mkdir -p $datadir
+
+    echo "# Env. var basis for benchmarks benchmarks."
```

Comment:
I have a 75% suspicion that you intended this for the `.env` file. Doesn't seem important enough to re-push for...